_config.yml

```diff
@@ -1 +0,0 @@
-theme: jekyll-theme-cayman
```

Comment:
Uhmm, waar is dit van? Maar even toevoegen in .gitignore

Reply:
Dit lijkt te komen vanuit de handleiding

Reply:
Done!